--- servod.c.orig	2019-12-14 21:20:25.017826132 +0000
+++ servod.c	2019-12-14 21:23:07.465887501 +0000
@@ -60,8 +60,8 @@
 #define DEFAULT_SERVO_MIN_US	500
 #define DEFAULT_SERVO_MAX_US	2500
 
-#define DEVFILE			"/dev/servoblaster"
-#define CFGFILE			"/dev/servoblaster-cfg"
+#define DEVFILE			"/servoblaster/servoblaster"
+#define CFGFILE			"/servoblaster/servoblaster-cfg"
 
 #define PAGE_SIZE		4096
 #define PAGE_SHIFT		12
